templates/corporate/upgrade.html

```diff
@@ -101,44 +132,41 @@ <h3>{{ _("License management") }}</h3>
 
                                 <div id="license-automatic-section">
                                     <p>
-                                        {% if free_trial_days %}
-                                        After the Free Trial, you&rsquo;ll be charged
+                                        {% if realm_on_free_trial %}
+                                        You&rsquo;ll initially be charged
                                         <b>$<span id="charged_amount"></span></b> for <b>{{ seat_count }}</b>
-                                        users (or more if you later add more users).<br>
-
-                                        We'll automatically charge you for additional licenses as users
-                                        are added, and remove licenses not in use at the end of each billing
-                                        period.
+                                        users (or more if you later add more users) at the end of free trial.<br>
                                         {% else %}
                                         You&rsquo;ll initially be charged
```

Comment:
I noticed we're kind of inconsistent with the usage of `&rsquo;` vs `'` (not just here, in some other files too). I don't know what's preferred (I think `&rsquo;` is typographically more correct, but what are the odds of us managing to be consistent with its usage) but I assume it's better to be consistent. Should we open an issue for discussing and fixing that?

Reply:
Yeah. We should probably open an issue or start a discussion in chat.zulip.org.